### PR TITLE
[policerorch]: Add PolicerOrch to bundle with mirror session

### PR DESCRIPTION
**Pull request by stcheng**
Now that we could create a policer for the mirror session to
throttle the mirroring traffic.

configuration:

POLICER|NAME:
 meter_type:packets|bytes
 mode:sr_tcm|tr_tcm|storm_control
 cir|DIGITS
 cbs|DIGITS
 pir|DIGITS
 pbs|DIGITS
 corlor_source:aware|blind
 red_action:drop
 yellow_action:drop
 green_action:drop

MIRROR_SESSION|NAME:
 policer:policer_name

Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

**Comment by lguohan** [comment]
can you update the swss-schema.md?

**Comment by stcheng** [comment]
@lguohan updated

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

File: orchagent/orchdaemon.cpp [hunk 1]
```diff
@@ -163,7 +165,7 @@ bool OrchDaemon::init()
      * when iterating ConsumerMap.
      * That is ensured implicitly by the order of map key, "LAG_TABLE" is smaller than "VLAN_TABLE" in lexicographic order.
      */
-    m_orchList = { gSwitchOrch, gCrmOrch, gBufferOrch, gPortsOrch, gIntfsOrch, gNeighOrch, gRouteOrch, copp_orch, tunnel_decap_orch, qos_orch, wm_orch };
+    m_orchList = { gSwitchOrch, gCrmOrch, gBufferOrch, gPortsOrch, gIntfsOrch, gNeighOrch, gRouteOrch, copp_orch, tunnel_decap_orch, qos_orch, wm_orch, policer_orch };
```
Comment (lguohan) [inline orchagent/orchdaemon.cpp]: >policer_orch  [](start = 152, length = 13)

is this the right sequence?

File: orchagent/policerorch.cpp [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+    return false;
+}
+
+bool PolicerOrch::increaseRefCount(const string &name)
+{
+    SWSS_LOG_ENTER();
+
+    if (!policerExists(name))
+    {
+        return false;
```
Comment (lguohan) [inline orchagent/policerorch.cpp]: warning message here?

File: orchagent/policerorch.cpp [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    }
+
+    ++m_policerRefCounts[name];
+
+    SWSS_LOG_NOTICE("Policer %s reference count is increased to %d",
```
Comment (lguohan) [inline orchagent/policerorch.cpp]: INFO level, does not look like NOTICE level.

File: orchagent/policerorch.cpp [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+    return true;
+}
+
+bool PolicerOrch::decreaseRefCount(const string &name)
+{
+    SWSS_LOG_ENTER();
+
+    if (!policerExists(name))
+    {
+        return false;
```
Comment (lguohan) [inline orchagent/policerorch.cpp]: warning message here.

File: orchagent/policerorch.cpp [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+    }
+
+    --m_policerRefCounts[name];
+
+    SWSS_LOG_NOTICE("Policer %s reference count is decreased to %d",
+            name.c_str(), m_policerRefCounts[name]);
```
Comment (lguohan) [inline orchagent/policerorch.cpp]: info level

File: orchagent/policerorch.cpp [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+            vector<sai_attribute_t> attrs;
+            bool meter_type = false, mode = false;
+
+            for (auto i = kfvFieldsValues(tuple).begin();
+                    i != kfvFieldsValues(tuple).end(); ++i)
+            {
+                auto field = to_upper(fvField(*i));
+                auto value = to_upper(fvValue(*i));
+
+                SWSS_LOG_INFO("attribute: %s value: %s", field.c_str(), value.c_str());
```
Comment (lguohan) [inline orchagent/policerorch.cpp]: debug level

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
retest this please

**Comment by stcheng** [comment]
```
test_mirror.py::TestMirror::test_MirrorAddRemove PASSED                  [ 49%]
test_mirror.py::TestMirror::test_MirrorToVlanAddRemove PASSED            [ 50%]
test_mirror.py::TestMirror::test_MirrorToLagAddRemove PASSED             [ 50%]
test_mirror.py::TestMirror::test_MirrorDestMoveVlan PASSED               [ 51%]
test_mirror.py::TestMirror::test_MirrorDestMoveLag PASSED                [ 52%]
test_mirror.py::TestMirror::test_AclBindMirror PASSED                    [ 53%]
test_mirror_ipv6_combined.py::TestMirror::test_AclBindMirrorV6Combined PASSED [ 54%]
test_mirror_ipv6_combined.py::TestMirror::test_AclBindMirrorV6Reorder1 PASSED [ 54%]
test_mirror_ipv6_combined.py::TestMirror::test_AclBindMirrorV6Reorder2 PASSED [ 55%]
test_mirror_ipv6_combined.py::TestMirror::test_AclBindMirrorV6WrongConfig PASSED [ 56%]
test_mirror_ipv6_separate.py::TestMirror::test_AclBindMirrorSeparated PASSED [ 57%]
test_mirror_ipv6_separate.py::TestMirror::test_AclBindMirrorV6Reorder1 PASSED [ 58%]
test_mirror_ipv6_separate.py::TestMirror::test_AclBindMirrorV6Reorder2 PASSED [ 58%]
test_mirror_ipv6_separate.py::TestMirror::test_AclBindMirrorV6WrongConfig PASSED [ 59%]
test_mirror_policer.py::TestMirror::test_MirrorPolicer PASSED            [ 60%]
test_mirror_policer.py::TestMirror::test_MirrorPolicerWithAcl PASSED     [ 61%]
```

**Pull request merged**